h2o-docs/src/product/mojo-quickstart.rst

```diff
@@ -609,6 +609,10 @@ The following code snippet shows how to download a MOJO from R and run the Print
 FAQ
 ~~~
 
+-  **Are MOJOs thread safe?**
+
+  All of H2O-3 MOJOs are threadsafe except for Deep Learning MOJOs. 
```

Comment:
can we change threadsafe to thread-safe

Reply:
Actually, it should be without the hyhen. Will update.